engine/core/src/main/java/org/datacleaner/metamodel/datahub/DataHubDeleteBuilder.java

```diff
@@ -25,11 +26,22 @@
 import org.apache.metamodel.query.FilterItem;
 import org.apache.metamodel.schema.Table;
 
-public class DataHubDeleteBuilder extends AbstractRowDeletionBuilder{
+public class DataHubDeleteBuilder extends AbstractRowDeletionBuilder {
+    private static final String GR_ID_COLUMN_NAME = "gr_id";
+    private static final String SOURCE_ID_COLUMN_NAME = "source_id";
+    private static final String SOURCE_NAME_COLUMN_NAME = "source_name";
+
+    enum RecordType {
```

Comment:
I guess you mean that we don't want the names of the columns hardcoded in the client? I checked the schema were are fetching from DataHub and that has a primary key property set to true for the gr_id (golden records) and the source_id and source_name (source records). So maybe we can use that to check the where clause. Would that be a good solution?